app.js

```diff
@@ -1,21 +1,25 @@
-document.addEventListener('DOMContentLoaded', getCount);
-let counter1 = 0;
-let counter2 = 0;
-document.querySelector('.my-pic1').addEventListener('click', function(){ 
-    counter1++;
-    localStorage.setItem('counter1',counter1);
-    document.getElementById('btn1').innerHTML= localStorage.getItem('counter1');
-});
+document.querySelector('.my-pic1').addEventListener('click', function(){Counter('btn1');});
+
+document.querySelector('.my-pic2').addEventListener('click', function(){Counter('btn2');});
 
-document.querySelector('.my-pic2').addEventListener('click', function(){
-    counter2++;
-    localStorage.setItem('counter2',counter2);
-    document.getElementById('btn2').innerHTML= localStorage.getItem('counter2');
+document.addEventListener('DOMContentLoaded', function(){
+      getCount();
 });
 
+function Counter(btn){
+    const currElem = document.getElementById(btn);
+    const currCount = parseInt(currElem.innerText);
+    localStorage.setItem(btn,currCount+1);
+    currElem.innerText= localStorage.getItem(btn);   
+}
+
 function getCount(){
-    document.getElementById('btn1').innerHTML= localStorage.getItem('counter1');
-    document.getElementById('btn2').innerHTML= localStorage.getItem('counter2');
-    counter1 = localStorage.getItem('counter1');
-    counter2 = localStorage.getItem('counter2');
-}
\ No newline at end of file
+    const storageVal1 = localStorage.getItem('btn1');
+    const value1 = !storageVal1 && isNan(storageVal1)?0:localStorage.getItem('btn1');

```

Comment:
these validations can be moved to a util function as well, better you can move all the storage utils to a different file and import that file to use those functions

Reply:
Created a new utility function and imported modules from there in the latest comitt. It has increased the readability also now.